modules/ROOT/pages/data-modeling.adoc

```diff
@@ -67,13 +77,18 @@ You should change any top-level user properties that have a key with a leading u
 * Wrapping them inside another object with a key that doesn't have a leading underscore.
 
 
+
+
+
+
```

Comment:
Lots of whitespace 😄 
(not a problem, it does help scan an asciidoc file, just made me wonder if there's anything about this in the style guide)